.github/workflows/build.yaml

```
@@ -0,0 +1,66 @@
+name: Build and push container image
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - '*'
+    branches:
+      - main
+
+env:
+  REGISTRY: quay.io
+  IMAGE_NAME: jumpstarter-dev/jumpstarter
+  QUAY_ORG: quay.io/jumpstarter-dev
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+      attestations: write
+      id-token: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get version
+        run: |
+          VERSION=$(git describe --long --tags)
```

```suggestion
          VERSION=$(git describe --tags)
```

This is an error I am fixing in other repositories (my mistake)

Otherwise for the v0.0.0 tag... it would still generate v0.0.0-0-g435914e